PL2/PL2-gui/src/main/java/nl/tudelft/pl2016gr2/gui/view/graph/DrawComparedGraphs.java

```diff
@@ -112,8 +122,11 @@ public void drawGraphs(ArrayList<NodePosition> topGraph,
     updateGraph();
   }
 
+  /**
+   * Update the graph by redrawing it.
+   */
   private void updateGraph() {
-    if (topGraph == null || bottomGraph == null) {
+    if (topGraphOrder == null || bottomGraphOrder == null) {
       return;
     }
```

Comment:
If there isn't any graph being shown in the graph view, it shouldn't be updated.